### PR TITLE
DOC-10704 Author content for Intra device sync

### PR DESCRIPTION
**Pull request by dinaabennett**
Changes for ticket and fixing misc typos

### CODE REVIEW

File: modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc [hunk 1]
```diff
@@ -10,24 +10,21 @@
...
-This might benefit, for example, a mobile application developer striving to minimize consumer wait time (arising from data setup) on initial install and launch of the application.
+{cbl-te} supports  pre-built databases. You can pre-load your app with data instead of syncing it from {sgw-te} during startup to minimize consumer wait time (arising from data setup) on initial install and launch of the application. 
 
-Avoiding an initial bulk sync will help reduce startup time and network transfer costs, leading to a better consumer experience.
-It is typically more efficient to download bulk data using the http/ftp stream  employed during the application installation than to install a smaller application bundle and then have to use a replicator to pull-in the bulk data.
+Avoiding an initial bulk sync reduces startup time and network transfer costs. It is typically more efficient to download bulk data using the http/ftp stream employed during the application installation than to install a smaller application bundle and then use a replicator to pull-in the bulk data.
```
Comment (franrob-projects) [inline modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc]: 'pull-in' I don't think this work fits the context. Maybe use just use 'pull in'

**Review by franrob-projects — APPROVED** [review]
I've made a suggestion _but_ at your preference. All looks good  👍

File: modules/ROOT/pages/_partials/commons/common-database.adoc [hunk 2]
```diff
@@ -45,11 +45,11 @@ The most basic structure is to use the single default scope with a single defaul
 .Couchbase Lite Examples
 include::partial$diagram-definitions.adoc[tag=couchbase-lite-example-structure]
 
-If a scope is not explicitly referenced in calls to the database, then scope is assumed to `_default`. Likewise, if no collection name is given then then the `_default` collection is assumed.
+If a scope is not explicitly referenced in calls to the database, then scope is assumed to `_default`. Likewise, if no collection name is given then the `_default` collection is assumed.
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-database.adoc]: Might as well quickfix to ventilate prose (e.g. put new sentence "Likewise..." on a new line)

File: modules/ROOT/pages/_partials/commons/common-database.adoc [hunk 3]
```diff
@@ -45,11 +45,11 @@ The most basic structure is to use the single default scope with a single defaul
...
 
 .Storing local configuration
 ****
-It's likely that you will not need to sync all the data related for a particular application, in which case you can set up a scope that syncs data, and a second stop that doesn't.
+You may not need to sync all the data related for a particular application. You can set up a scope that syncs data, and a second scope that doesn't.
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-database.adoc:54]: New line for new sentence.

File: modules/ROOT/pages/_partials/commons/common-dbreplica.adoc [hunk 4]
```diff
@@ -22,9 +22,9 @@ include::{root-partials}_show_page_header_block.adoc[]
 
 include::{root-partials}block-caveats.adoc[tag=enterprise-only]
 
-Couchbase Lite supports replication between two local databases.
+Couchbase Lite supports replication between two local databases at the database, scope, or collection level.
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-dbreplica.adoc:25]: "databases at the database" looks a bit odd? Hmm, probably fine, just looked weird viewing the edit out of context.

File: modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc [hunk 5]
```diff
@@ -10,24 +10,21 @@
 == Overview
 
 
-{cbl-te}_'s_ support for pre-built databases means you can pre-load your app with data instead of syncing it down from {sgw-te} during startup.
-This might benefit, for example, a mobile application developer striving to minimize consumer wait time (arising from data setup) on initial install and launch of the application.
+{cbl-te} supports  pre-built databases. You can pre-load your app with data instead of syncing it from {sgw-te} during startup to minimize consumer wait time (arising from data setup) on initial install and launch of the application.
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc:13]: New line for new sentence.

Comment (osfameron) [reply modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc:13]: Not urgent for now, but in near future we could look at replacing `{cbl-te}` and friends either with a more readable label or, even 😱 typing the whole term out...

File: modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc (hunk 1 above)
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc]: new line for new sentence

Comment (osfameron) [reply modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc]: (again sorry, that's not something you introduced, but just as you edited it 😄 anyway, you get the point)

File: modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc [hunk 6]
```diff
@@ -10,24 +10,21 @@
...
 
-Avoiding an initial bulk sync will help reduce startup time and network transfer costs, leading to a better consumer experience.
-It is typically more efficient to download bulk data using the http/ftp stream  employed during the application installation than to install a smaller application bundle and then have to use a replicator to pull-in the bulk data.
+Avoiding an initial bulk sync reduces startup time and network transfer costs. It is typically more efficient to download bulk data using the http/ftp stream employed during the application installation than to install a smaller application bundle and then use a replicator to pull-in the bulk data.
 
-This prepackaging of the data typically applies to public/shared, non-user specific data that is mostly static.
-Even if the data is not static, you can still benefit from preloading it and only syncing the changed documents on startup
+Pre-loaded data is typically public/shared, non-user-specific data that is static. Even if the data is not static, you can still benefit from preloading it and only syncing the changed documents on startup.
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc:19]: new line etc.

"mostly static" seems to be different from "static".
I realise the caveat on next line kinda explains that, but calling it out isn't terrible.

**Review by osfameron — APPROVED** [review]
Couple of minor things, mostly grumbling about ventilated prose, sorry 😸 

Lots of good clear rewordings 👍

File: modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc (hunk 6 above)
Comment (dinaabennett) [reply modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc:19]: mostly is a wishy washy word - we talk about non-static data in the next sentence.

File: modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc [hunk 7]
```diff
@@ -10,24 +10,21 @@
...
-This might benefit, for example, a mobile application developer striving to minimize consumer wait time (arising from data setup) on initial install and launch of the application.
+{cbl-te} supports  pre-built databases. You can pre-load your app with data instead of syncing it from {sgw-te} during startup to minimize consumer wait time (arising from data setup) on initial install and launch of the application. 
 
-Avoiding an initial bulk sync will help reduce startup time and network transfer costs, leading to a better consumer experience.
-It is typically more efficient to download bulk data using the http/ftp stream  employed during the application installation than to install a smaller application bundle and then have to use a replicator to pull-in the bulk data.
+Avoiding an initial bulk sync reduces startup time and network transfer costs. It is typically more efficient to download bulk data using the http/ftp stream employed during the application installation than to install a smaller application bundle and then use a replicator to pull in the bulk data.
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc]: ventilate

**Review by osfameron — APPROVED** [review]
👍

**Review by franrob-projects — APPROVED** [review]
LGTM

**Pull request merged**